tests/unit/test-util/test.adb.js

```diff
@@ -236,6 +237,52 @@ describe('utils/adb', () => {
       sinon.assert.calledOnce(adb.util.readAll);
       assert.deepEqual(packages, ['org.mozilla.fennec', 'org.mozilla.firefox']);
     });
+
+    it('resolves only the given firefox APK', async () => {
+      const adb = getFakeADBKit({
+        adbClient: {
+          shell: sinon.spy(() => Promise.resolve('')),
+        },
+        adbkitUtil: {
+          readAll: sinon.spy(() => {
+            return Promise.resolve(new Buffer(fakeADBPackageList));
```

Comment:
The `Buffer` constructor is deprecated. Please use `Buffer.from(fakeADBPackageList)` instead.